docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,342 @@
+=================================
+Check Neutron objects overlapping
+=================================
+
+One of the most tricky parts of Cloud migration is :ref:`networking`. During
+migration to the non-empty cloud there can be plenty of conflicts and overlaps
+from the Neutron side. And it is better to know such issues before the actual
+migration starts. For detecting these kinds of problems :dfn:`CheckNetworks`
+action exists.
+
+This action is defined as ``check_networks`` task in the ``preparation``
+section of :ref:`scenario <scenario-files-config>`. There are several examples
+of migration scenarios with this action can be found in the
+:file:`CloudFerry/scenario` folder, like :file:`cold_migrate.yaml` (all-in-one
+migration scenario, tries to migrate all the openstack objects in a single run)
+or :file:`stages/0_prechecks.yaml` (runs all initial checks, which allows to
+spot most of the problems in configuration).
+
+.. important::
+
+    It is highly recommended to run this check before every :ref:`networking`
+    to detect any possible conflicts and resolve it before actual migration
```

Comment:
It is a reference to another documentation and it will be shown up as `Neutron networking migration`.